### PR TITLE
ci: automate-labels for dependabot PRs

### LINKED ISSUE #1464: ci: automate-labels fails for dependabot PRs
See https://github.com/celestiaorg/celestia-app/actions/runs/4383348272/jobs/7673482212

### PR DESCRIPTION
**Pull request by amityadav0**
Automate-labels for dependabot PRs

Closes #1464

### CODE REVIEW

File: .github/workflows/issue-label-automation.yml
Comment (rootulp) [inline .github/workflows/issue-label-automation.yml:42]:
```suggestion
      # If a PR was created by dependabot add the `bot` label.
```

**Review by rootulp — COMMENTED** [review]
LGTM after a comment change! Thanks for the contribution

**Review by staheri14 — COMMENTED** [review]
Thanks for the contribution, looks good to me too! though, I refer it to the code owners to give the final approval.

**Review by rootulp — APPROVED** [review]

**Pull request merged**
